hack/setup-kind-in-ci.sh

```diff
@@ -61,11 +61,17 @@ TARGET_BRANCH="${TARGET_BRANCH:-master}"
 if [ -z "${ISTIO_VERSION}" ]; then
   if [ "${TARGET_BRANCH}" == "v1.48" ]; then
     ISTIO_VERSION="1.13.0"
+  elif [ "${TARGET_BRANCH}" == "v1.57" ]; then
+    ISTIO_VERSION="1.15.0"
+  elif [ "${TARGET_BRANCH}" == "v1.65" ]; then
+    ISTIO_VERSION="1.17.0"
+  elif [ "${TARGET_BRANCH}" == "v1.73" ]; then
+    ISTIO_VERSION="1.18.0"
   fi
 fi
 
 KIND_NODE_IMAGE=""
-if [ "${ISTIO_VERSION}" == "1.13.0" ]; then
+if [[ "${ISTIO_VERSION}" == "1.13.0" || "${ISTIO_VERSION}" == "1.15.0" || "${ISTIO_VERSION}" == "1.17.0" ]]; then
```

Comment:
this is fine, but FYI, you dont need to use the 2-bracket solution if you use the `-o` syntax like:
`if [ "${ISTIO_VERSION}" == "1.13.0" -o "${ISTIO_VERSION}" == "1.15.0" -o "${ISTIO_VERSION}" == "1.17.0" ]; then`

Reply:
ok, for the next time what is better to use? 2-bracket or `-o` option?